### PR TITLE
Add TagName support

### PR DESCRIPTION
**Pull request by nxm**
You can access the tagname by atom fields
`subset = FindSubnetByTagName(root, atom.Tbody)`

Or you can match it by string with lookup function

`atom.Lookup([]byte("tbody"))`

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -67,6 +69,11 @@ func main() {
 		if len(subset) == 0 {
 			log.Fatalf("Unable to find an element with the class '%s'\n", *byClass)
 		}
+	} else if *byTagName != "" {
+		subset = restify.FindSubnetByTagName(root, atom.Lookup([]byte(*byTagName)))
```
Comment (itzg) [inline cmd/main.go:73]: It should be named `FindSubsetByTagName` for consistency. README will also need to be tweaked.

Comment (itzg) [inline cmd/main.go:73]: Please follow the pattern of the others where second arg should only be `*byTagName`. The atom details should be encapsulated inside the function.

File: go.mod [hunk 2]
```diff
@@ -1,4 +1,4 @@
-module github.com/itzg/restify
+module github.com/nxm/restify
```
Comment (itzg) [inline go.mod:1]: Leftover from fork

**Review by itzg — CHANGES_REQUESTED** [review]
Great addition!

**Comment by nxm** [comment]
Thanks for CR ❤️

**Review by itzg — APPROVED** [review]
Thanks for the updates

**Comment by itzg** [comment]
This is now released https://github.com/itzg/restify/releases/tag/1.6.0

**Pull request merged**